Security/JWTAuthenticator.php

```diff
@@ -40,6 +40,9 @@ public function __construct(JWTUserProvider $userProvider, KernelInterface $kern
     public function createToken(Request $request, $providerKey)
     {
         $jwt = $request->query->get('jwt');
+        if(!$jwt && $request->headers->has("authorization") && (count($authorizationHeaderArray = explode(" ",$request->headers->get("authorization"))) > 1)) {
```

Comment:
Could you please remove assignment from the condition and use by-index access to the array elements instead of `list()`? It is for better readability